src/viz/index.ts

```diff
@@ -40,24 +42,15 @@ export const createFromTemplate = async (answers: Answers): Promise<number> => {
     }
   });
 
-  await util.spinnify('Building & pushing dev deployment...', async () => {
-    if (answers.yarn) {
-      await util.exec('yarn run build:dev', {cwd: projectPath}, false);
-      await util.exec('yarn run push:dev', {cwd: projectPath}, false);
-    } else {
-      await util.exec('npm run build:dev', {cwd: projectPath}, false);
-      await util.exec('npm run push:dev', {cwd: projectPath}, false);
-    }
-  });
+  const colorCd = chalk.black.bgCyan.bold('cd ' + projectName);
+  const colorNpm = chalk.black.bgCyan.bold('npm run start');
```

Comment:
Check out how this is done in the connectors section. Since we allow yarn as the build tool via a --yarn flag.